doc/Android.md

```diff
@@ -0,0 +1,109 @@
+**Document based on Minetest `5.6.1` Android build**
+# Minetest: Android version
+Minetest for Android are the same Minetest on Windows but compiled for Android, not seperate version
+## Controls
+The Android port doesn't support everything you can do on PC due to the
+limited capabilities of common devices. What can be done is described
+below:
+
+While you're playing the game normally (that is, no menu or inventory is
+shown), the following controls are available:
+* Look around: touch screen and slide finger
+* double tap: place a node
+* long tap: dig node or use the holding item
+* touch buttons: press button
+* Buttons:
```

Comment:
```suggestion
In regular gameplay, the controls are as follows:

* Look around: swiping
* double tap: performs a "place" action which includes placing nodes
* long tap: performs a "punch" action which includes digging nodes

**Buttons:**
```

Reply:
i think i will create a seperate doc for technical stuff, like instead of "Press back" i will use "KEY_BACK"

also Android have some "legacy" arrow key exist since Android device have physical arrow button, is Minetest use this as move key too?

Reply:
Which arrow button/key are you referring to?
Regardless on whether it is an internal workaround or an actual key, there's no key remapping on Android, hence  I don't think that it needs further elaboration in this document.

Reply:
oh that key are called dpad keycode keyevent